packages/vscode-extension/src/telemetry/extTelemetryEvents.ts

```diff
@@ -40,6 +40,9 @@ export enum TelemetryEvent {
   UpdatePreviewManifestStart = "update-preview-manifest-start",
   UpdatePreviewManifest = "update-preview-manifest",
 
+  ConnectExistingApiStart = "connect-exist-api-start",
+  ConnectExistingApi = "aconnect-exist-api",
```

Comment:
typo, by the way, can we name the event as connect-existing-api?

Reply:
thanks!